src/test/java/org/kainos/ea/integrationTests/JobRoleControllerTest.java

```diff
@@ -0,0 +1,24 @@
+package org.kainos.ea.integrationTests;
+
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.ea.model.JobRoleRequest;
+import org.kainos.ea.trueApplication;
+import org.kainos.ea.trueConfiguration;
+
+import java.util.List;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class JobRoleControllerTest {
+    static final DropwizardAppExtension<trueConfiguration> APP = new DropwizardAppExtension<>(trueApplication.class, null, new ResourceConfigurationSourceProvider());
+
+    @Test
+    void getJobRoles_shouldReturnListOfJobRoles() {
```

Comment:
Are we able to assert the status code of the response as well? Should be a 200? https://http.cat/200

Reply:
Done